sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -144,16 +144,16 @@ namespace Azure { namespace Core { namespace Http {
      * @param httpMethod HTTP method.
      * @param url URL.
      * @param bodyStream #Azure::Core::IO::BodyStream.
-     * @param downloadViaStream A boolean value indicating whether download should happen via
-     * stream.
+     * @param noBufferDownload A boolean value indicating whether download should not use a buffer
+     * for the response but provide a body stream instead.
      */
     explicit Request(
         HttpMethod httpMethod,
         Url url,
         Azure::Core::IO::BodyStream* bodyStream,
-        bool downloadViaStream)
+        bool noBufferDownload)
```

Comment:
`bool bufferedDownload` reads a lot better.

Reply:
Yes, I will change the entire logic.